pages/events/summer-school.vue

```diff
@@ -157,6 +142,7 @@ interface AgendaSlot {
   },
   async asyncData (context: Context) {
     const eventData = await context.$content('events/summer-school-2021').fetch() as IContentDocument
+    const faqData = await context.$content('events/summer-school-2021-page/faq').fetch() as IContentDocument
```

Comment:
Nice to have: Add the `.catch()` method to return an error page in case the fetching fails instead of failing silently or when the JS runtime fails down the line.